docs/contributing.rst

```diff
@@ -233,17 +233,43 @@ Making a new release
 
 A checklist for creating, testing, and distributing a new version.
 
-#. Choose a version number, e.g. ``VERSION=3.3.0``.
-#. Create a new branch, e.g. ``git switch -c release-$VERSION``.
-#. Run ``tox -e changelog -- --version $VERSION`` to build
-   :file:`docs/changelog.rst`.
-#. Commit and open a pull request for review.
-#. Merge the pull request, and ensure the `GitHub Actions`_ build passes.
-#. Create a new git tag with ``git tag -m "Release v$VERSION" $VERSION``.
-#. Push the new tag with ``git push upstream $VERSION``.
-#. Watch the release in `GitHub Actions`_.
-#. Send announcement email to `distutils-sig mailing list`_ and celebrate.
+#. Choose a version number, and create a new branch
 
+   .. code-block:: console
+
+      VERSION=3.4.2
+
+      git switch -c release-$VERSION
+
+#. Update :file:`docs/changelog.rst`
+
+   .. code-block:: console
+
+      tox -e changelog -- --version $VERSION
+
+      git commit -am "Update changelog for $VERSION"
+
+#. Open a pull request for review
+
+#. Merge the pull request, and ensure the `GitHub Actions`_ build passes
+
+#. Create a new git tag for the version
+
+   .. code-block:: console
+
+      git switch main
+
+      git pull --ff-only upstream main
+
+      git tag -m "Release v$VERSION" $VERSION
```

Comment:
When did we stop signing and annotating tags?

Reply:
This instruction changed in https://github.com/pypa/twine/pull/544, and there was further discussion in https://github.com/pypa/twine/pull/667#pullrequestreview-435896146 and https://github.com/jaraco/keyring/issues/427.

Reply:
Okay, I just need to ensure I never try to release because my muscle memory will end up in a signed tag which will confuse folks. Too bad 1 person gets to dictate the process here rather than following community best practices and trying to meet downstream redistributors somewhere in the middle